language/de.json

```diff
@@ -0,0 +1,372 @@
+{
+	"Preview": "Vorschau",
+	"Gallery": "Galerie",
+	"Generate": "Generieren",
+	"Skip": "Überspringen",
+	"Stop": "Stop",
+	"Input Image": "Eingabebild",
+	"Advanced": "Erweitert",
+	"Upscale or Variation": "Vergrößern oder Abwandeln",
+	"Image Prompt": "Bild Prompt",
+	"Inpaint or Outpaint (beta)": "Inpainting oder Outpainting (beta)",
+	"Drag above image to here": "Ziehen Sie das obere Bild hierher",
+	"Upscale or Variation:": "Vergrößern oder Abwandeln:",
+	"Disabled": "Deaktiviert",
+	"Vary (Subtle)": "Variieren (Subtil)",
+	"Vary (Strong)": "Variieren (Stark)",
+	"Upscale (1.5x)": "Vergrößern (1.5x)",
+	"Upscale (2x)": "Vergrößern (2x)",
+	"Upscale (Fast 2x)": "Vergrößern (Schnell 2x)",
+	"\ud83d\udcd4 Document": "\uD83D\uDCD4 Dokument",
+	"Image": "Bild",
+	"Stop At": "Anhalten bei",
+	"Weight": "Gewicht",
+	"Type": "Typ",
+	"PyraCanny": "PyraCanny",
+	"CPDS": "CPDS",
+	"* \"Image Prompt\" is powered by Fooocus Image Mixture Engine (v1.0.1).": "* \"Bild Prompt\" wird von Fooocus Bildmischmotor (v1.0.1) angetrieben.",
+	"The scaler multiplied to positive ADM (use 1.0 to disable).": "Skalierung multipliziert mit positivem ADM (1.0 verwenden, um zu deaktivieren).",
+	"The scaler multiplied to negative ADM (use 1.0 to disable).": "Skalierung multipliziert mit negativem ADM (1.0 verwenden, um zu deaktivieren).",
+	"When to end the guidance from positive/negative ADM.": "Wann die Anleitung von positivem/negativem ADM beendet werden soll.",
```

Comment:
"guidance" is kind difficult to translate. Suggestion: Leitfaden, Leitlinie

Reply:
This suggestion has now been implemented